yb-voyager/src/srcdb/postgres.go

```diff
@@ -415,6 +416,12 @@ func GetAbsPathOfPGCommand(cmd string) (string, error) {
 		currVersion := strings.Fields(string(stdout))[2]
 
 		if version.CompareSimple(currVersion, PG_COMMAND_VERSION[cmd]) >= 0 {
+			// Check if the version of the command is less than the source DB version
+			if version.CompareSimple(currVersion, sourceDBVersion) < 0 {
+				err = fmt.Errorf("version of %v (%v) is less than the source DB version (%v)", cmd, currVersion, sourceDBVersion)
```

Comment:
we should NOT error out here. It's possible we find another version of pg_dump that satisfies our conditions. We should `continue`.

Reply:
Yeah, agree with Aneesh.
Check should be outside the for loop, to check and return err it couldn't find a suitable version.